### DISCUSSION TITLE
Valvole termostatiche RV05

### DISCUSSION THREAD

**Discussion by vpomax** [indomus/forum#861] [HUB: Home Assistant]
Qualcuno sta usando queste valvole termostatiche zigbee per termosifoni in Home Assistant?

[http://eurosolution.com/temp/rv05.png](url)

**Comment by snakuzzo** [comment]
Ci sono già discussioni aperte su valvole termostatiche zigbee...usa la funzione "Search" ... [esempio](https://github.com/indomus/forum/discussions?discussions_q=termostatiche)
